packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -190,18 +191,30 @@ class Tooltip extends StatefulWidget {
   /// The length of time that a pointer must hover over a tooltip's widget
   /// before the tooltip will be shown.
   ///
-  /// Once the pointer leaves the widget, the tooltip will immediately
-  /// disappear.
-  ///
   /// Defaults to 0 milliseconds (tooltips are shown immediately upon hover).
   final Duration? waitDuration;
 
   /// The length of time that the tooltip will be shown after a long press
-  /// is released.
+  /// is released or mouse pointer exits the widget.
   ///
-  /// Defaults to 1.5 seconds.
+  /// Defaults to 1.5 seconds for long press released or 0.1 seconds for mouse
+  /// pointer exits the widget.
   final Duration? showDuration;
 
+  static final Set<_TooltipState> _openedToolTips = <_TooltipState>{};
+
+  /// Dismiss the most recently shown tooltip.
+  ///
+  /// This method returns true if it successfully dismisses a tooltip. It returns
+  /// false if there is no tooltip shown on the screen.
+  static bool dismissToolTip() {
+    if (_openedToolTips.isNotEmpty) {
+      _openedToolTips.first._hideTooltip(immediately: true);
```

Comment:
Why just the first one?  Why not hide them all?

Reply:
sounds good